build.gradle

```diff
@@ -308,6 +309,7 @@ project('spring-integration-core') {
 		compile("io.fastjson:boon:$boonVersion", optional)
 		compile("com.esotericsoftware:kryo-shaded:$kryoShadedVersion", optional)
 
+		testCompile project(":spring-integration-test")
```

Comment:
No need to have it here.
It's just enough to fix the common dependencies pattern above.
Will fix on merge, if I don't find anything critical.